applications/nrf_desktop/configuration/nrf52840dk_nrf52840/app_ZDebug.conf

```diff
@@ -93,7 +93,7 @@ CONFIG_FLASH_MAP=y
 CONFIG_NVS=y
 
 CONFIG_SETTINGS=y
-# CONFIG_SETTINGS_NVS=y
+CONFIG_SETTINGS_NVS=y
```

Comment:
that's not a typo...
Anyway it won't change a thing

Reply:
It could change a lot if settings backend used by default changes in Zephyr

Reply:
That I know, so I don't question fixing the configuration. I meant it won't change the current behavior.

Reply:
And since it's a `small mistake in text`, maybe we could call it a `typo` (I am pretty sure it was not intentional to add comment here :) )

Reply:
I don't mind the name - changing commit message is not worth rerunning CI :)